librz/include/rz_syscall.h

```diff
@@ -24,10 +24,16 @@ typedef struct rz_syscall_item_t {
 	char *sargs;
 } RzSyscallItem;
 
-typedef struct rz_syscall_port_t {
-	int port;
-	const char *name;
-} RzSyscallPort;
+typedef struct rz_sysreg_item_t {
+	char *type;
+	char *name;
+	char *comment;
+	ut64 address;
```

Comment:
The `address` is probably redundant, since it's the key in the hashtable.

Reply:
Actually, `address` is one of the values, right? I had changed the format of the SDB, resembling the one from types.

https://github.com/rizinorg/rizin/pull/1160/files#diff-cbadf4a774e67c5b7b27c4c6355eb978b19137269cdcb46b55789d0c7300a187R5-R7

Reply:
Hey, you are right about `address` being redundant. 

But, I'm not sure about how I'm going to parse it inside `get_sysreg_type` and bring it back to `sdb_load_sysregs`, which is where it's getting inserted into the hashtable.

Reply:
Combined the functionality of `get_sysreg_type` into `sdb_load_sysregs` and eliminated the redundant variable.